kalibro_client/miscellaneous/native_metric.py

```diff
@@ -3,8 +3,8 @@
 
 class NativeMetric(Metric):
     def __init__(self, languages=None, metric_collector_name=None, *init_args, **init_kwargs):
-        init_kwargs['type'] = 'NativeMetricSnapshot'
-        super(NativeMetric, self).__init__(*init_args, **init_kwargs)
+        tpe = init_kwargs.pop('type', 'NativeMetricSnapshot')
```

Comment:
`type` is a built-in function and it's bad practice to shadow it. `type_` would work as well (I have no strong preference for either choice).